airflow/providers/microsoft/azure/triggers/wasb.py

```diff
@@ -86,3 +86,80 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:
                         await asyncio.sleep(self.poke_interval)
         except Exception as e:
             yield TriggerEvent({"status": "error", "message": str(e)})
+
+
+class WasbPrefixSensorTrigger(BaseTrigger):
+    """
+    WasbPrefixSensorTrigger is fired as a deferred class with params to run the task in trigger.
+    It checks for the existence of a blob with the given prefix in the provided container.
+
+    :param container_name: name of the container in which the blob should be searched for
+    :param prefix: prefix of the blob to check existence for
+    :param include: specifies one or more additional datasets to include in the
+            response. Options include: ``snapshots``, ``metadata``, ``uncommittedblobs``,
+            ``copy`, ``deleted``
+    :param delimiter: filters objects based on the delimiter (for e.g '.csv')
+    :param wasb_conn_id: the connection identifier for connecting to Azure WASB
+    :param poke_interval:  polling period in seconds to check for the status
+    :param public_read: whether an anonymous public read access should be used. Default is False
+    """
+
+    def __init__(
+        self,
+        container_name: str,
+        prefix: str,
+        include: list[str] | None = None,
+        delimiter: str | None = "/",
```

Comment:
```suggestion
        delimiter: str = "/",
```
Seems like this should be the actual typing?

Reply:
fixed